### PR TITLE
chore: update core and most of the models & fields

### PR DESCRIPTION
**Pull request by elpiel**

### CODE REVIEW

File: src/main/proto/stremio/core/runtime/event.proto.orig [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+    PlayerStopped player_stopped = 27;
+    PlayerEnded player_ended = 28;
+    TraktPlaying trakt_playing = 29;
+    TraktPaused trakt_paused = 30;
+    MagnetParsed magnet_parsed = 31;
+    TorrentParsed torrent_parsed = 32;
+    PlayingOnDevice playing_on_device = 33;
+>>>>>>> 0b39eef (chore: Add missing runtime events for notifications)
+    Error error = 100;
+  }
```
Comment (sleeyax) [inline src/main/proto/stremio/core/runtime/event.proto.orig:76]: Remnant of a merge conflict? Perhaps delete the file, I don't see why you'd want to keep this `.orig` copy around.

File: src/main/proto/stremio/core/types/video.proto [hunk 2]
```diff
@@ -15,9 +15,11 @@ message Video {
   optional string thumbnail = 5;
   repeated Stream streams = 6;
   optional SeriesInfo seriesInfo = 7;
-  required bool upcoming = 8;
-  required bool watched = 9;
-  required bool current_video = 10;
+  // TODO: Add trailer streams
```
Comment (sleeyax) [inline src/main/proto/stremio/core/types/video.proto:18]: Remove `TODO`.

**Review by sleeyax — CHANGES_REQUESTED** [review]

File: src/commonMain/rust/bridge/continue_watching_item.rs [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+use url::Url;
+
+use crate::protobuf::stremio::core::models;
+use crate::protobuf::stremio::core::types;
+
+use super::ToProtobuf;
+
+impl ToProtobuf<models::ContinueWatchingItem, (Option<&StreamsItem>, Option<&Url>, &Settings)> for Item {
+    fn to_protobuf(&self, args: &(Option<&StreamsItem>, Option<&Url>, &Settings)) -> models::ContinueWatchingItem {
+        // Option<&StreamsItem>,
```
Comment (TheBeastLT) [inline src/commonMain/rust/bridge/continue_watching_item.rs:14]: commented code?

File: src/main/proto/stremio/core/runtime/action_meta_details.proto [hunk 4]
```diff
@@ -6,14 +6,16 @@ option java_package = "com.stremio.core.runtime.msg";
 
 import "google/protobuf/empty.proto";
 
+import "stremio/core/types/video.proto";
```
Comment (TheBeastLT) [inline src/main/proto/stremio/core/runtime/action_meta_details.proto:9]: no need for new line after each import

File: src/main/proto/stremio/core/runtime/event.proto.orig (hunk 1 above)
Comment (elpiel) [reply src/main/proto/stremio/core/runtime/event.proto.orig:76]: yes, it's by mistake, thanks for noticing!

File: src/commonMain/rust/bridge/continue_watching_item.rs [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+use stremio_core::types::streams::StreamsItem;
+use url::Url;
+
+use crate::protobuf::stremio::core::models;
+use crate::protobuf::stremio::core::types;
+
+use super::ToProtobuf;
+
+impl ToProtobuf<models::ContinueWatchingItem, (Option<&StreamsItem>, Option<&Url>, &Settings)> for Item {
+    fn to_protobuf(&self, args: &(Option<&StreamsItem>, Option<&Url>, &Settings)) -> models::ContinueWatchingItem {
```
[the remark this answers is not in the document]
Comment (elpiel) [reply src/commonMain/rust/bridge/continue_watching_item.rs:13]: This is one of the places that needs implementing as there's a `todo!()` in it. We don't always have a `video id` so we can extract the correct `SteamsItem` from the `StreamsBucket`.

File: src/commonMain/rust/model/fields/meta_details.rs [hunk 6]
```diff
@@ -65,6 +76,8 @@ impl
             thumbnail: self.thumbnail.clone(),
             streams: self.streams.to_protobuf(&(None, *addon_name, None, None)),
             series_info: self.series_info.to_protobuf(&()),
+            // trailer_streams: self.trailer_streams.to_protobuf(&()),
+            trailer_streams: todo!(),
```
[the remark this answers is not in the document]
Comment (elpiel) [reply src/commonMain/rust/model/fields/meta_details.rs:80]: will remove

[the remark this answers is not in the document]
Comment (elpiel) [reply src/commonMain/rust/model/fields/meta_details.rs:80]: removed

**Pull request merged**